liqd_product/apps/users/templates/liqd_product_users/indicator.html

```diff
@@ -1,19 +1,20 @@
 {% load i18n %}
 
 {% if request.user.is_authenticated %}
-    <div class="dropdown dropdown--right">
+    <div class="dropdown userindicator__dropdown">
```

Comment:
without dropdown--right the menu is out of the browsers viewport
But with dropdown--right, the styling is broken in the mobile page as the button will float to the right :(
not sure how to fix that. we proabably have to copy the styles from the dropdown--right to the userindicator--dropdown

Note: that i just fixed the css in meinberlin, now dropdown--right is working again.